packages/playground-ui/src/ds/components/Badge/badge.stories.tsx

```
@@ -0,0 +1,76 @@
+    variant: 'default',
+  },
+};
+
+export const Success: Story = {
+  args: {
+    children: 'Success',
+    variant: 'success',
+  },
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)